app/account/page.tsx

```diff
@@ -0,0 +1,29 @@
+"use client";
+
+import React from "react";
+import { isAuthenticated } from "@/firebase/authentication";
+import { redirect } from "next/navigation";
+import { getCurrentUser } from "@/firebase/authentication";
+import { SignOutButton } from "@/components/buttons";
+
+const ProfilePage: React.FC = () => {
+  if (!isAuthenticated()) {
+    redirect("/login");
+  }
+
+  const user = getCurrentUser();
+
+  return (
+    <main className="flex min-h-screen items-center justify-around">
```

Comment:
Nope... We should have a `<main>` element in the root layout. No need to have a separate `<main>` for each page/component

Reply:
Agree. Has been fixed